product.go

```diff
@@ -81,7 +105,7 @@ type Product struct {
 	IngredientsFromPalmOilNumber                int            `json:"ingredients_from_palm_oil_n"`
 	IngredientsFromPalmOilTags                  []interface{}  `json:"ingredients_from_palm_oil_tags"`
 	IngredientsIdsDebug                         []string       `json:"ingredients_ids_debug"`
-	IngredientsN                                int            `json:"ingredients_n"`
+	IngredientsN                                StringInt      `json:"ingredients_n"`
```

Comment:
perhaps use `encoding/json.Number` instead of our own baked solution?

[encoding/json.Number](https://godoc.org/encoding/json#Number)